<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogoContable</class>
 <widget class="QDialog" name="CatalogoContable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catalogo Contable</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="5" colspan="4">
    <widget class="QTreeWidget" name="Catalogo">
     <property name="maximumSize">
      <size>
       <width>580</width>
       <height>680</height>
      </size>
     </property>
     <property name="locale">
      <locale language="Spanish" country="ElSalvador"/>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Codigo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Catalogo de Cuentas</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="4" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nombre de la Cuenta</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="codigoCuenta">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Debe</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="debe">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="6" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Haber</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="haber">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Saldo Deudor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="saldoDeudor">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="6" colspan="3">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Saldo Acreedor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="saldoAcreedor">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="4" colspan="5">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Descripci√≥n</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="crearRubro">
     <property name="text">
      <string>Crear Rubro</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="crearCuenta">
     <property name="text">
      <string>Crear Cuenta</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="crearEspecifico">
     <property name="text">
      <string>Crear Especifico</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="transaccion">
     <property name="text">
      <string>Crear Transaccion</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QToolButton" name="pushButton_3">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/exit.png</normaloff>:/iconos/exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QToolButton" name="btmSeleccionar">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/button_ok.png</normaloff>:/iconos/button_ok.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QToolButton" name="botonBuscar">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../recursosIconos.qrc">
       <normaloff>:/Iconos/iconos/buscar.png</normaloff>:/Iconos/iconos/buscar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
  <include location="../recursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
